CHANGES.rst

```diff
@@ -8,6 +8,9 @@ Changelog
   This may require adaptations in testsSupport messages with line separation ``\r\n``
   (`#35 <https://github.com/zopefoundation/Products.MailHost/issues/35>`_).
 
+- Bugfix Tests
+  (`#37 <https://github.com/zopefoundation/Products.MailHost/issues/37>`_).
+
```

Comment:
I think this entry is not necessary because you are fixing tests which where broken after the last release.